src/charm.py

```diff
@@ -87,6 +155,9 @@ def _filter_lines(line: str) -> bool:
 
         Args:
             line: A log line from the log file.
+
+        Returns:
+            A Boolean that indicates if the line must be included or not.
```

Comment:
I wouldn't include the return type as part of the docs string, we already have type hints :)